src/components/visualizations/chart/highcharts/plugins/test/getTickPositions.spec.ts

```diff
@@ -517,6 +517,40 @@ describe('get tick positions for dual axes chart', () => {
                 });
             });
 
+            it('left tick position is empty with undefined min/max,' +
```

Comment:
Let's use newer style of describing tests:
1) test description starts with the word `should` in order to improve readability => `it('should...`
2) please keep description on single line and rather add exception for linter `// eslint-disable-next-line`